### PR TITLE
Added options to hide certain UI elements

### PR DESCRIPTION
**Pull request by timblakely**
Defaults to showing the full display. Also fixes noimplicitany error in layout.ts=>Handler

### CODE REVIEW

File: src/neuroglancer/viewer.ts [hunk 1]
```diff
@@ -31,7 +31,10 @@ import {TrackableBoolean} from 'neuroglancer/trackable_boolean';
 import {TrackableValue} from 'neuroglancer/trackable_value';
 import {RefCounted} from 'neuroglancer/util/disposable';
 import {vec3} from 'neuroglancer/util/geom';
-import {GlobalKeyboardShortcutHandler, KeySequenceMap} from 'neuroglancer/util/keyboard_shortcut_handler';
```
Comment (jbms) [inline src/neuroglancer/viewer.ts]: Can you revert (i.e. via rebasing) unrelated formatting changes?

File: src/neuroglancer/viewer.ts [hunk 2]
```diff
@@ -55,6 +58,21 @@ export function validateLayoutName(obj: any) {
...
 
+export interface UIOptions {
+  showLayerPanel?: boolean;
+  showHelpButton?: boolean;
+  showLocation?: boolean;
+  showLayerDialog?: boolean;
+}
+export interface ViewerOptions extends UIOptions {}
+
+let DefaultViewerOptions: ViewerOptions = {
```
Comment (jbms) [inline src/neuroglancer/viewer.ts]: Make this const, and call it defaultViewerOptions instead.

File: src/neuroglancer/widget/shader_code_widget.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@
 
 import 'codemirror/addon/lint/lint.js';
 
-import CodeMirror from 'codemirror';
```
Comment (jbms) [inline src/neuroglancer/widget/shader_code_widget.ts]: Please also revert this unrelated change.  Depending on how the typings are defined, how webpack is configured, and in what mode typescript is run, one or the other import style is needed.  However, I believe the existing style is what works best with the current neuroglancer configuration.

**Comment by timblakely** [comment]
Sorry about the formatting changes! <3 clang-format + formatOnSave... maybe a little bit too much. PTAL.
